src/main/java/hu/elte/eserial/Getter.java

```diff
@@ -0,0 +1,61 @@
+package hu.elte.eserial;
```

Comment:
This class looks pretty out of place, could we have something like a `model` package for it?

Reply:
It's been moved to the model package.